client/capstone-app/src/components/MapWithRoute.jsx

```diff
@@ -16,7 +16,7 @@ const MapWithRoute = ({ routeData, directions, centerCoordinates}) => {
   };
 
   return (
-    <MapContainer center={[-90.071533, 29.951065]} zoom={15} style={{ height: '60vh', width: '100vh' }}>
+    <MapContainer center={centerCoordinates} zoom={13} style={{ height: '60vh', width: '100vh' }}>
       <TileLayer
```

Comment:
Just confirming, is this commit addressing the comments from https://github.com/desolagiwa/metaU-capstone/pull/4? Next time, it could be good to say that explicitly in the commit message.

Reply:
Yes! Thanks for the heads up!